backend/battle/forms.py

```diff
@@ -94,19 +94,24 @@ class Meta:
 
     def clean(self):
         cleaned_data = super().clean()
-        pokemon_1 = self.cleaned_data.get('pokemon_1')
-        pokemon_2 = self.cleaned_data.get('pokemon_2')
-        pokemon_3 = self.cleaned_data.get('pokemon_3')
-        obj_trainer = cleaned_data['trainer']
 
         if 'battle' not in cleaned_data:
             raise forms.ValidationError('ERROR: Select a valid battle')
 
         obj_battle = cleaned_data['battle']
 
-        if not pokemon_1 or not pokemon_2 or not pokemon_3:
+        if 'trainer' not in cleaned_data:
```

Comment:
I feel that this should not be at the form level. Please assume here that the trainer is always filled because of the battle

Reply:
great!